### PR TITLE
Fix adqs

### PR DESCRIPTION
**Pull request by sebbacon**

### CODE REVIEW

File: openprescribing/frontend/tests/commands/test_import_adqs.py [hunk 1]
```diff
@@ -1,40 +1,31 @@
...
+            mock_query.query.call_args[0][0])
 
-        p = Presentation.objects.get(bnf_code='0401020K0AAACAC')
-        self.assertEqual(p.adq, None)
-        self.assertEqual(p.adq_unit, None)
+        # Check we set adq_per_quantity according to the bigquery results
+        p = Presentation.objects.get(bnf_code='0202010F0AAAAAA')
+        self.assertEqual(p.adq_per_quantity, 0.333)
+        p = Presentation.objects.get(bnf_code='0204000I0JKKKAL')
+        self.assertEqual(p.adq_per_quantity, None)
```
Comment (inglesp) [inline openprescribing/frontend/tests/commands/test_import_adqs.py]: Good to see we're testing both that a change has been made when expected _and_ that a change hasn't been made when not expected.

File: openprescribing/frontend/models.py [hunk 2]
```diff
@@ -349,12 +349,9 @@ class Presentation(models.Model):
                                 validators=[isAlphaNumeric])
     name = models.CharField(max_length=200)
     is_generic = models.NullBooleanField(default=None)
-    active_quantity = models.FloatField(null=True, blank=True)
-    adq = models.FloatField(null=True, blank=True)
-    adq_unit = models.CharField(max_length=10, null=True, blank=True)
     is_current = models.BooleanField(default=True)
-    percent_of_adq = models.FloatField(null=True, blank=True)
     replaced_by = models.ForeignKey('self', null=True, blank=True)
+    adq_per_quantity = models.FloatField(null=True, blank=True)
```
Comment (inglesp) [inline openprescribing/frontend/models.py]: Please could you add a comment explaining how to think about `adq_per_quantity`?

**Comment by sebbacon** [comment]
On deploy:

* [ ] Run migrations
* [ ] Run the new management comment to compute ADQs
* [ ] Recalculate `ktt9_uti_antibiotics` measure and check
* [ ] Point rich at that measure to show how he should use ADQs in the future

**Comment by sebbacon** [comment]
I also had to fix some code I'd mocked wrong, and delete and recreate the `presentations` table in bigquery (q.v. #1082)

**Pull request merged**